docs/prover/cli/options.md

```diff
@@ -258,12 +255,15 @@ Whenever you want to use a Solidity compiler executable with a non-default name.
 
 **What does it do?**  
 Gets a list of arguments to pass to the Solidity compiler. The arguments will be passed as is, without any formatting, in the same order.  
+arguments that are defined as standalone flags cannot be defined in `--solc_args`, including the solc flags  `--optimize`, 
```

Comment:
`solc` fails spell checking; you should either use "the Solidity compiler" or
```
`solc`
```

Reply:
done